### ISSUE TITLE
"Please direct your browser to" link gives an error

### ISSUE THREAD

**Issue by kevinw** [Byron/google-apis-rs#190]
After installing the calendar api with

```
cargo install google-calendar3-cli
```

and trying the tool with

```
calendar3 calendar-list list
```

I got the expected message saying

```
Please direct your browser to https://accounts.google.com/o/oauth2/auth?scope=https://www.googleapis.com/auth/calendar.readonly&redirect_uri=http://localhost:54324&response_type=code&client_id=XXXXXXXXX-9ngf7o4dhs0dka470npqvor6dc5lqb9b.apps.googleusercontent.com and follow the instructions displayed there.
```

(I removed the client_id.) But loading the url in my browser, I get an error from google:

```

401. That’s an error.

Error: deleted_client

The OAuth client was deleted.

Request Details
That’s all we know.
```

Does the URL need to be updated?

**Comment by Byron** [comment]
The default clients that the library comes with have been deleted by google. For now the only way to get this to work is to provide your own application secret.

This is [documented for each API](http://byron.github.io/google-apis-rs/google_adexchangeseller2_cli/index.html#application-secrets), the link on the left is just an example.

These days it would probably be best to not have a default at all, which should be much less confusing.

**Comment by Byron** [comment]
I am closing this issue as since then a lot has changed. All crates are now `async`, available as v2.0 on crates.io and build against the latest API spec.
Please let me know in the comments if the issue is still relevant though.
